spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -6,13 +6,7 @@
   let(:headers) { { 'HTTP_CAPKEY' => Settings.API_KEY, 'CONTENT_TYPE' => 'application/json' } }
   let(:publication) { create :publication }
   let(:author) { create :author }
-  # These authorships can be merged with any contribution hash
-  let(:cap_author_hash) do
-    { cap_profile_id: author.cap_profile_id }
-  end
-  let(:sul_author_hash) do
-    { sul_author_id: author.id }
-  end
+  let(:sul_author_hash) { Hash[sul_author_id: author.id] }
```

Comment:
I guess `cap_author_hash` moved somewhere else.  The idea of grouping it all here is to see how it is based on the author.  Apart from one-liners, don't care much for this change, but it doesn't have to be a blocker, unless cap_author_hash disappeared (prob not).